### ISSUE TITLE
[BUG] Kevlar & Helmet

### ISSUE THREAD

**Issue by crashzk** [LordFetznschaedl/CS2Retake#11]
There is a round where there is a team where the players only come with Kevlar and another team with Kevlar & Helmet, even the weapon kits are all configured with `"Kevlar": "KevlarHelmet"`.

Weapon kit configuration I'm using:
```
[
    {
      "KitName": "M4A4",
      "PrimaryWeapon": "weapon_m4a1",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": true,
      "Kevlar": "KevlarHelmet",
      "Team": 3,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "M4A1-S",
      "PrimaryWeapon": "weapon_m4a1_silencer",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": true,
      "Kevlar": "KevlarHelmet",
      "Team": 3,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "FAMAS",
      "PrimaryWeapon": "weapon_famas",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": true,
      "Kevlar": "KevlarHelmet",
      "Team": 3,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "AUG",
      "PrimaryWeapon": "weapon_aug",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": true,
      "Kevlar": "KevlarHelmet",
      "Team": 3,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "AK-47",
      "PrimaryWeapon": "weapon_ak47",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": false,
      "Kevlar": "KevlarHelmet",
      "Team": 2,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "Galil AR",
      "PrimaryWeapon": "weapon_galilar",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": false,
      "Kevlar": "KevlarHelmet",
      "Team": 2,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "SG 553",
      "PrimaryWeapon": "weapon_sg556",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": false,
      "Kevlar": "KevlarHelmet",
      "Team": 2,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
	{
      "KitName": "SSG 08",
      "PrimaryWeapon": "weapon_ssg08",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": false,
      "Kevlar": "KevlarHelmet",
      "Team": 2,
      "RoundType": "Undefined",
      "KitLimit": -1
    },
    {
      "KitName": "CT AWP",
      "PrimaryWeapon": "weapon_awp",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": true,
      "Kevlar": "KevlarHelmet",
      "Team": 3,
      "RoundType": "Undefined",
      "KitLimit": 1
    },
    {
      "KitName": "T AWP",
      "PrimaryWeapon": "weapon_awp",
      "SecondaryWeapon": "weapon_deagle",
      "DefuseKit": false,
      "Kevlar": "KevlarHelmet",
      "Team": 2,
      "RoundType": "Undefined",
      "KitLimit": 1
    }
  ]
```

**Comment by crashzk** [comment]
I believe documentation of how the weapon kit system works would be interesting.

**Comment by LordFetznschaedl** [comment]
Hello @crashzk 

Thank you for mentioning this bug. 
I know this bug exists that sometimes (better said too often) kits are defined using KevlarHelmet but then spawn without helmet.

This bug is logged and I will try to solve this.

Thank you

**Comment by BKOKAN** [comment]
i just added round types on kits and made specific kits for pistols. In those i just specify round types and it works just fine but the issue with that is pistol round that is supposed to be 800$ i get random nades and it goes over the 800$ pistol limit

**Comment by LordFetznschaedl** [comment]
Hello @crashzk ,

The issue with the KevlarHelmet should now be resolved with release v1.0.3-alpha.

Regards,
Robert
